src/fidesops/api/v1/urn_registry.py

```diff
@@ -88,3 +87,8 @@
 
 # Health URL
 HEALTH = "/health"
+
+# DRP
+DRP = "/drp"
+DRP_EXERCISE = DRP + "/exercise"
+DRP_STATUS = DRP + "/status"
```

Comment:
NIT: can we store URLs as absolute strings so that they're easily `cmd + f`-able?

Reply:
should we eventually update the rest of the `urn_registry` to be `cmd + f`-able in the same way?